### PR TITLE
Sync grafan dashboard panel for pipeline service

### PR DESCRIPTION
**Pull request by divyansh42**

### CODE REVIEW

File: components/monitoring/grafana/base/dashboards/pipeline-service/grafana-config.yaml [hunk 1]
```diff
@@ -152,7 +152,7 @@ data:
             {
               "editorMode": "code",
               "exemplar": true,
-              "expr": "sum(increase(pipeline_service_schedule_overhead_percentage_sum{status='succeded'}[30m])) / sum(increase(pipeline_service_schedule_overhead_percentage_count{status='succeded'}[30m]))",
+              "expr": "sum(increase(pipeline_service_schedule_overhead_percentage_sum{status='succeded'}[2m])) / sum(increase(pipeline_service_schedule_overhead_percentage_count{status='succeded'}[2m]))",
```
[the remark this answers is not in the document]
Comment (gabemontero) [reply components/monitoring/grafana/base/dashboards/pipeline-service/grafana-config.yaml:155]: I know why but this is a good, educational exercise for you @divyansh42  .... common practices for developers for questions like this, regardless of scenario, is to look at `git blame` and git histories for the files in question.

The files in question are this one, the placement-service-dashboard.json file in the pipeline-service repo, and the kustomization.yaml file in the same directory as this file.

See if you can sort it out when you are back to work, and we'll talk on slack or scrum where we are both available.

Bottom line, we want to to with the 2m changes here, but you need to cultivate these investigative skills, so I want you to know how to sort our questions like this.

[the remark this answers is not in the document]
Comment (gabemontero) [reply components/monitoring/grafana/base/dashboards/pipeline-service/grafana-config.yaml:155]: @divyansh42 and I got and slack and I coached him through the above process.

Summary: infra-deployment was pointing to a pipeline-service level before I changed the intervals from 30m to 2m

**Comment by gabemontero** [comment]
Yeah I've tried a couple of times now @divyansh42 and the "Pipeline Service" dashboard is missing from the appstudio-grafana folder

I've never debugged the grafana operator before, but I took a guess wrt grepping and found this error log in the grafana-operator-controller-manager:

```
2024-08-14T16:10:32Z	ERROR	GrafanaDashboardReconciler	error fetching dashboard	{"controller": "grafanadashboard", "controllerGroup": "grafana.integreatly.org", "controllerKind": "GrafanaDashboard", "GrafanaDashboard": {"name":"grafana-dashboard-pipeline-service","namespace":"appstudio-grafana"}, "namespace": "appstudio-grafana", "name": "grafana-dashboard-pipeline-service", "reconcileID": "29c4d7b3-790c-49e9-8873-bd17b0300fcb", "dashboard": "grafana-dashboard-pipeline-service", "error": "ConfigMap \"grafana-dashboard-pipeline-service\" not found"}
```

So that is definitely related to what we are changing here.

I'll do a local deploy off of main branch to make sure the issue does not occur there as well.  Also, I confirmed that the Konflux clusters are at the latest version of infra deployments for grafana, and the panels are in the list there.

But maybe that is because the ConfigMap already exists?  I suspect that may be the case, and the namespaces got messed up when I pulled the yaml from pipeline-service via `kustomize build` and your PR here is a victim of that.

@enarha FYI

**Comment by gabemontero** [comment]
ok @divyansh42 after https://github.com/redhat-appstudio/infra-deployments/pull/4342 merges you can rebase this PR and we should be able to see your panel updates when testing this against a local/personally provisioned cluster.

**Comment by gabemontero** [comment]
ok https://github.com/redhat-appstudio/infra-deployments/pull/4342 has merged

**Comment by divyansh42** [comment]
@gabemontero I rebased the PR. Please take a look.

**Review by gabemontero — APPROVED** [review]
/lgtm

tested this branch locally and looks good

File: components/monitoring/grafana/base/dashboards/pipeline-service/grafana-config.yaml [hunk 2]
```diff
@@ -421,6 +421,68 @@ data:
...
+                    "color": "green",
+                    "value": null
+                  },
+                  {
+                    "color": "red",
+                    "value": 30
+                  }
+                ]
+              },
+              "unit": "percent"
```
Comment (gabemontero) [inline components/monitoring/grafana/base/dashboards/pipeline-service/grafana-config.yaml:446]: congrats on your first infra-deployment merge @divyansh42 !!

that said, I noticed this minor error only after merge, and not during my review / testing of your change

rather than percent, this line ^^ should be 

`"unit": "none"`

craft a quick PR when you come in on Friday and I'll tag for merge when I get in.

Comment (gabemontero) [reply components/monitoring/grafana/base/dashboards/pipeline-service/grafana-config.yaml:446]: PR https://github.com/redhat-appstudio/infra-deployments/pull/4354 is up
